### PR TITLE
Add OnPlayerStreamIn.md (ID) OnPlayerStreamOut.md (ID) OnPlayerStateChange (ID) OnPlayerTakeDamage (ID)

### PR DESCRIPTION
**Pull request by AthallahDzaki**
Tolong Di Koreksi Bos :smile:

### CODE REVIEW

File: docs/translations/id/scripting/callbacks/OnPlayerStateChange.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: OnPlayerSpawn
```
Comment (Eiikirin) [inline docs/translations/id/scripting/callbacks/OnPlayerStateChange.md:2]: Title salah.

File: docs/translations/id/scripting/callbacks/OnPlayerStateChange.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Returns
+
+Selalu terpanggil pertama di filterscripts.
+
+## Contoh
+```c
+public OnPlayerStateChange(playerid, newstate, oldstate)
+{
+    if (oldstate == PLAYER_STATE_ONFOOT && newstate == PLAYER_STATE_DRIVER) // Player entered a vehicle as a driver
```
Comment (Eiikirin) [inline docs/translations/id/scripting/callbacks/OnPlayerStateChange.md:27]: Komentar belum diterjemahkan

File: docs/translations/id/scripting/callbacks/OnPlayerStreamIn.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
```
Comment (Eiikirin) [inline docs/translations/id/scripting/callbacks/OnPlayerStreamIn.md:1]: Coba lihat contoh ini: https://github.com/openmultiplayer/web/blob/master/docs/translations/id/scripting/callbacks/OnActorStreamIn.md

File: docs/translations/id/scripting/callbacks/OnPlayerStreamOut.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+---
```
Comment (Eiikirin) [inline docs/translations/id/scripting/callbacks/OnPlayerStreamOut.md:1]: Coba lihat contoh ini: https://github.com/openmultiplayer/web/blob/master/docs/translations/id/scripting/callbacks/OnActorStreamIn.md

File: docs/translations/id/scripting/callbacks/OnPlayerStreamOut.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+title: OnPlayerStreamOut
+description: Callback ini akan terpanggil ketika pemain menjauh dari pemain lain.
+tags: ["player"]
+---
+
+<VersionWarnID name='Callback' version='SA-MP 0.3a' />
```
Comment (Eiikirin) [inline docs/translations/id/scripting/callbacks/OnPlayerStreamOut.md:7]: Coba dibaca: 
https://github.com/openmultiplayer/wiki/issues/263#issuecomment-720026425

File: docs/translations/id/scripting/callbacks/OnPlayerTakeDamage.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
+---
+title: OnPlayerTakeDamage
+description: Callback ini terpanggil ketika pemain menerima damage.
+tags: ["player"]
+---
+
+<VersionWarn name='callback' version='SA-MP 0.3d' />
```
Comment (Eiikirin) [inline docs/translations/id/scripting/callbacks/OnPlayerTakeDamage.md:7]: Coba dibaca:
https://github.com/openmultiplayer/wiki/issues/263#issuecomment-720026425

File: docs/translations/id/scripting/callbacks/OnPlayerStreamIn.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    return 1;
+}
+```
+
+## Catatan
+
+<TipNPCCallbacks />
+
+## Fungsi Terkait
+// Tidak Ada
```
Comment (Eiikirin) [inline docs/translations/id/scripting/callbacks/OnPlayerStreamIn.md:38]: Tidak perlu pakai //tidak ada

File: docs/translations/id/scripting/callbacks/OnPlayerStreamOut.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+    return 1;
+}
+```
+
+## Catatan
+
+<TipNPCCallbacks />
+
+## Fungsi Terkait
+// Tidak Ada
```
Comment (Eiikirin) [inline docs/translations/id/scripting/callbacks/OnPlayerStreamOut.md:38]: Tidak perlu pakai //tidak ada

File: docs/translations/id/scripting/callbacks/OnPlayerTakeDamage.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+:::
+
+:::warning
+
+GetPlayerHealth dan GetPlayerArmour akan memberikan amount lama dari pemain sebelum memanggil callback ini. Selalu cek jika issuerid valid sebelum melakukan array index.
+
+:::
+
+## Related Functions
+// Tidak Ada
```
Comment (Eiikirin) [inline docs/translations/id/scripting/callbacks/OnPlayerTakeDamage.md:82]: Tidak perlu pakai //tidak ada

**Review by Eiikirin — CHANGES_REQUESTED** [review]
Ada beberapa yang kurang tepat, coba di perbaiki lagi.

**Comment by AthallahDzaki** [comment]
@Eiikirin https://github.com/openmultiplayer/web/blame/master/docs/translations/id/scripting/callbacks/OnActorStreamOut.md#L15 coba cek salah Translate kak

**Comment by Eiikirin** [comment]
Eh kok malah edit komen ya akwoakwoakw

**Comment by Eiikirin** [comment]
fokus sama PR, gak usah kemana²

**Comment by Eiikirin** [comment]
[MERGED] to #121

**Pull request closed without merging**